addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -966,6 +984,9 @@ const getOrCreateSpan = (node, ancestors) => {
 }
 const removeFormat = (node, formatSpec) => {
     node = closestElement(node);
+    if (formatSpec["removeClass"]) {
+        formatSpec.removeClass(node);
+    }
```

Comment:
Why ? You're postulating that a class name is a format, so you should use the existing api of formats, not add a new entry just for this usecase.

Reply:
You are right, I will change that :+1: